src/ngraph/opsets/opset0_tbl.hpp

```diff
@@ -193,7 +193,6 @@ NGRAPH_OP(Softmax, ngraph::op)
 NGRAPH_OP(SoftmaxCrossEntropy, ngraph::op)
 NGRAPH_OP(SoftmaxCrossEntropyBackprop, ngraph::op)
 NGRAPH_OP(SpaceToDepth, ngraph::op)
-NGRAPH_OP(Split, ngraph::op)
```

Comment:
Why `Split` has been removed from `opset0`?

Reply:
That's my mistake, thanks, I'll revert it.